lp-core-platform/lp-cp-xwiki-implementation/src/main/java/eu/learnpad/core/impl/me/XwikiController.java

```diff
@@ -180,6 +185,9 @@ public MVResults checkModelSetVerification(String verificationProcessId)
 					// TODO Auto-generated catch block
 					e.printStackTrace();
 				}
+				// Notify QM about a new model set imported
+				String fooModelContent = "-ask-to-jean-how-to-retraive-the-modelset-";
+				this.qm.importModelSet(toReturn.getModelSetId(), toReturn.getType(), fooModelContent.getBytes());
```

Comment:
You should look into other components that do exactly the same operation. I guess that the following piece of code should help you.
https://github.com/LearnPAd/learnpad/blob/master/lp-core-platform/lp-cp-xwiki-implementation/src/main/java/eu/learnpad/core/impl/cw/XwikiController.java#L131